.github/workflows/test.yaml

```diff
@@ -46,3 +46,45 @@ jobs:
 
       - name: Test
         run: poetry run pytest
+
+  package: # only on latest Ubuntu and Python
+    runs-on: ubuntu-latest
+    name: Test packaging wheel and Conda
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: "x64"
+
+      - name: Set Java version
+        run: python scripts/set_versions.py
+
+      - uses: joschi/setup-jdk@v2 # We must use to the official action once https://github.com/actions/setup-java/pull/97 is closed
+        with:
+          java-version: ${{ env.MAJOR_JAVA_VERSION }}
+          java-package: jdk
+          architecture: x64
+
+      - name: Install poetry
+        run: pip3 install poetry
+
+      - name: Install python dependencies
+        run: poetry install
+
+      - name: Build JDK
+        run: poetry run python scripts/build_jdk.py
+
+      - name: Build wheel
+        env:
+          JDK4PY_BUILD_PLATFORM: ubuntu-latest
+        run: poetry run python setup.py bdist_wheel
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          channels: conda-forge
+          conda-build-version: 3.20.0
+
+      - name: Build Conda package on ubuntu and latest Python
+        run: conda build --output-folder dist . --no-anaconda-upload --debug
```

Comment:
Missing empty line at the end of the file ? (not sure if it is usefull but github always highlight that)